lib/watir.rb

```diff
@@ -46,11 +46,12 @@ def always_locate?
     end
 
     def always_locate_message
-      Watir.logger.warn <<-EOS
+      msg <<-EOS
```

Comment:
This will throw an "undefined method `msg' for Watir:Module" exception due to the missing assignment operator. It will also format a bit funny due to the return carriages:

~~~~~~~~
2018-06-06 23:03:10 WARN Watir [:always_locate, :deprecations] Watir#always_locate is deprecated; elements are always cached and will always
be re-located if they go stale before use.
Use Element#stale? or Element#wait_until(&:stale?) if needed for flow control.
~~~~~~~~

Would suggest:

~~~~~~~~
def always_locate_message
  msg = <<-EOS.gsub("\n", " ")
Watir#always_locate is deprecated; elements are always cached and will always
be re-located if they go stale before use.
Use Element#stale? or Element#wait_until(&:stale?) if needed for flow control.
  EOS
  Watir.logger.warn msg, ids: [:always_locate, :deprecations]
end
~~~~~~~~